coverage.go

```diff
@@ -139,6 +139,8 @@ func (l Parser) processLcov() (Report, error) {
 		}
 
 	}
+	// Add last file
+	report.Files = append(report.Files, fileReport)
```

Comment:
While it would be unusual, I suspect it is technically legal to have an lcov file with no files listed. If that is the case this should probably be:
```
                         if fileReport.Path != "" {
				report.Files = append(report.Files, fileReport)
			}
```